contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.0.0--3.1.0.sql

```diff
@@ -31,6 +31,36 @@ end
 $$
 LANGUAGE plpgsql;
 
+CREATE OR REPLACE FUNCTION sys.babelfish_update_server_collation_name() RETURNS VOID
```

Comment:
As discussed offline, we do not need it in 3_X at all. Because we only allow upgrade from 14.6 to 15 and so on. And the upgrade script from 2.3.0 (14.6) to 3.0.0 contains that info. So we do not need to add in any such upgrade script. Also, as Cx is in 3_X they can only to 4_X and beyond for which we have already updated the upgrade script.

Hence closing the PR.